src/babel-plugin-transform-sx-tailwind/src/index.js

```diff
@@ -1,9 +1,10 @@
 // @flow
 
-require('@babel/register')({
-  ignore: [/node_modules\/(?!@adeira)/],
-  rootMode: 'upward',
-});
+// prettier-ignore
+require('@babel/register')({ // @x-shipit-disable
```

Comment:
Should it really work this way? I guess Shipit isn’t involved into NPM publishing.

Reply:
Haha, yeah, that is a good point! 🤦

Reply:
@michalsanger Resolved here: https://github.com/adeira/universe/pull/1236 - sorry for the complications 🙈